shell/components/nav/WindowManager/ContainerLogs.vue

```diff
@@ -592,44 +583,17 @@ export default {
         ref="body"
         :class="{'logs-container': true, 'open': isOpen, 'closed': !isOpen, 'show-times': timestamps && filtered.length, 'wrap-lines': wrap}"
       >
-        <table
-          class="fixed"
-          cellpadding="0"
-          cellspacing="0"
-        >
-          <tbody class="logs-body">
-            <template v-if="filtered.length">
-              <tr
-                v-for="line in filtered"
-                :key="line.id"
-              >
-                <td
-                  :key="line.id + '-time'"
-                  class="time"
-                  v-html="format(line.time)"
-                />
-                <td
-                  :key="line.id + '-msg'"
-                  class="msg"
-                  v-html="line.msg"
-                />
-              </tr>
-            </template>
-            <tr v-else-if="search">
-              <td
-                v-t="'wm.containerLogs.noMatch'"
-                colspan="2"
-                class="msg text-muted"
-              />
-            </tr>
-            <tr
-              v-else
-              v-t="'wm.containerLogs.noData'"
-              colspan="2"
-              class="msg text-muted"
-            />
-          </tbody>
-        </table>
+        <VirtualList
+          v-show="filtered.length"
+          ref="virtualList"
+          data-key="id"
+          :data-sources="filtered"
+          :data-component="logItem"
```

Comment:
NOTE: I found this usage of `data-component` quite interesting.. I can't say that I've seen passing a component as a prop before, but I suppose this is an implementation decision of how we are expected to interact with the new library.

Reply:
Yeah I thought it was a little strange, but the library doesn't offer an alternative.